flocker/common/version.py

```diff
@@ -137,7 +137,8 @@ def get_installable_version(version):
     installed (CLI and node).
     """
     parsed_version = _parse_version(version)
-    return parsed_version.installable_release
+    returned_version = parsed_version.installable_release
+    return returned_version
```

Comment:
:question: Looks like an unnecessary change. Maybe there was some debug code in here during development?